src/detect-itype.c

```diff
@@ -221,14 +221,13 @@ static bool PrefilterITypeIsPrefilterable(const Signature *s)
 static int DetectITypeParseTest01(void)
 {
     DetectU8Data *itd = NULL;
-    int result = 0;
     itd = DetectITypeParse(NULL, "8");
```

Comment:
no need to update this PR, but in future PRs can you update such tests to declare `itd` on the line where it is set? So:
```c
DetectU8Data *itd = DetectITypeParse(...
```